propulate/islands.py

```diff
@@ -87,6 +89,9 @@ def __init__(
             The path where checkpoints are loaded from and stored to. Default is the current working directory.
         ranks_per_worker : int, optional
             The number of ranks per worker. Default is 1.
+        surrogate_factory : Callable[[], Surrogate], optional
+           Function that returns a new instance of a Surrogate model.
+           Only used when loss_fn is a generator function.
```

Comment:
```suggestion
           Function that returns a new instance of a ``Surrogate`` model.
           Only used when ``loss_fn`` is a generator function.
```
I added a different style for code things in your docstrings so they will be rendered accordingly in the automatically generated API references on the ReadTheDocs page.